tests/fixtures/coderunnerbuilder.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+
+"""
+This is a test preview generator builder.
+It executes the given file as a python code. The generated preview text contains
+the output of this code.
+This is unsafe and it should not be used in production.
+To enable this builder, put it in the builder folder.
+"""
+
+import sys
+import typing
+
+from preview_generator.extension import mimetypes_storage
+from preview_generator.preview.generic_preview import PreviewBuilder
+
+mimetypes_storage.add_type("application/x-preview-generator-test", ".runpy")
```

Comment:
Not needed here, but don't forget it's not the standard way to add new mimetype in a builder.